instructure/canvas-ios
# Offline grades

refs: MBL-16765
affects: Student
release note: none

test plan:
- Sync the grades tab for offline usage.
- Turn off internet on the device.
- Go to grades page.
- Page content should display the current grading period and grades should match the online version.
- Grading periods filter is disabled in offline mode.
- If grading period filter is modified in online mode then it will corrupt the offline data and it won't display the total grade, display assignments for the period selected in online mode instead of the last one while displaying the title of the last grading period.

## Checklist
